src/docs/src/install/freebsd.rst

```diff
@@ -16,71 +16,93 @@
+    couchdb3_enable="NO"
+    couchdb3_user="couchdb"
+    couchdb3_erl_flags="-couch_ini /usr/local/libexec/couchdb3/etc/default.ini /usr/local/etc/couchdb3/local.ini"
+    couchdb3_chdir="/var/db/couchdb3"
 
-After enabling the couchdb rc service use the following command to start CouchDB::
+After enabling the ``couchdb3`` service (by setting
+``couchdb3_enable`` to ``"YES"``), use the following command to start
+CouchDB::
 
-    /usr/local/etc/rc.d/couchdb start
+    service couchdb3 start
 
-This script responds to the arguments `start`, `stop`, `status`, `rcvar` etc..
+This script responds to the arguments `start`, `stop`, `status`,
+`rcvar` etc.  If the service is not yet enabled in ``rc.conf``, use
+``onestart`` to start it up ad-hoc.
 
-The start script will also use settings from the following config files:
+The service will also use settings from the following config files:
 
-- /usr/local/etc/couchdb/default.ini
-- /usr/local/etc/couchdb/local.ini
+- ``/usr/local/libexec/couchdb3/etc/default.ini``
+- ``/usr/local/etc/couchdb3/local.ini``
 
-Administrators should use ``default.ini`` as reference and only modify the
-``local.ini`` file.
+Administrators should use ``default.ini`` as a reference and only
+modify the ``local.ini`` file.
 
 Post install
-------------
-**Your installation is not complete. Be sure to complete the**
-:ref:`Setup <setup>` **steps for a single node or clustered installation.**
+============
 
-In case the install script fails to install a non-interactive user "couchdb" to
-be used for the database, the user needs to be created manually:
+**The installation is not complete. Be sure to complete the**
+:ref:`Setup <setup>` **steps for a single node or clustered
+installation.**
 
-I used the ``pw`` command to add a user "couchdb" in group "couchdb":
+Also note that the port will probably show some messages after the
+installation happened.  Make note of these instructions, although they
+can be found `in the ports tree`_ for later reference.
 
-.. code-block:: text
+In case the install script fails to install a non-interactive user
+``couchdb`` to be used for the database, the user needs to be created
+manually.
 
-    pw user add couchdb
-    pw user mod couchdb -c 'CouchDB, time to relax' -s /usr/sbin/nologin -d /var/lib/couchdb
-    pw group add couchdb
+Use the ``pw`` command from the base system to add a user ``couchdb``
+in group ``couchdb``::
 
-The user is added to ``/etc/passwd`` and should look similar to the following:
+    # pw useradd couchdb
+    # pw usermod couchdb -c 'CouchDB Account' -s /usr/sbin/nologin -d /var/empty
+    # pw groupadd couchdb
+    # pw groupmod couchdb -m couchdb
 
-.. code-block:: text
+Then the user is added to ``/etc/passwd`` and the group to
+``/etc/groups``.  These could be verified in the following way::
 
-    shell#  grep couchdb /etc/passwd
-    couchdb:*:1013:1013:Couchdb, time to relax:/var/lib/couchdb/:/usr/sbin/nologin
+    $ grep couchdb /etc/passwd
```

Comment:
oh dear how long has `/var/lib/couchdb` been here and I didn't notice?

IMHO we could remove this entire section unless you feel it's useful.

Reply:
I do not have any personal preference.  I am fine with losing it.